src/geode/model/representation/builder/brep_builder.cpp

```diff
@@ -81,6 +81,48 @@ namespace geode
         return mapping;
     }
 
+    void BRepBuilder::replace_components_meshes_by_others(
+        BRep&& other, const ModelCopyMapping& mapping )
+    {
+        for( const auto& in2out :
+            mapping.at( Corner3D::component_type_static() ).in2out_map() )
+        {
+            this->unregister_mesh_component( brep_.corner( in2out.first ) );
```

Comment:
Why unregister the corner? It is still here

Reply:
To unregister the mesh component from the VertexIdentifier, since its mesh will be crushed by the new one, and the unique vertices added at the end of the function

Reply:
why not using update_corner_mesh ?